src/arctic3d/modules/interface.py

```diff
@@ -187,12 +191,21 @@ def get_interface_residues(uniprot_id, out_uniprot_string, out_pdb_string):
     out_pdb_set = parse_out_pdb(out_pdb_string)
 
     interface_dict = {}
-    url = f"{INTERFACE_URL}/{uniprot_id}"
-    try:
-        interface_api_data = make_request(url, None)
-    except Exception as e:
-        log.warning(f"Could not make InterfaceResidues request for {uniprot_id}, {e}")
-        return interface_dict
+    if not interface_data:
+        url = f"{INTERFACE_URL}/{uniprot_id}"
+        try:
+            interface_api_data = make_request(url, None)
+        except Exception as e:
+            log.warning(
+                f"Could not make InterfaceResidues request for {uniprot_id}, {e}"
+            )
+            return interface_dict
+    else:
+        try:
+            interface_api_data = jsonpickle.decode(open(interface_data, "r").read())
```

Comment:
I think `jsonpickle.decode` transforms the .json in a python object, by the looks of it you need a dictionary in this step, is that right?

Reply:
precisely, `interface_api_data` is the dictionary directly converted from the json file. Exactly the same output you would get from a standard https request.